tests/components/tradfri/common.py

```diff
@@ -32,45 +43,77 @@ async def setup_integration(hass):
     return entry
 
 
-def modify_state(
-    state: dict[str, Any], partial_state: dict[str, Any]
-) -> dict[str, Any]:
-    """Modify a state with a partial state."""
-    for key, value in partial_state.items():
-        if isinstance(value, list):
-            for index, item in enumerate(value):
-                state[key][index] = modify_state(state[key][index], item)
-        elif isinstance(value, dict):
-            state[key] = modify_state(state[key], value)
-        else:
-            state[key] = value
+def register_device(
```

Comment:
could this become methods on the CommandStore ?

Reply:
Could the CommandStore already be initialized with the gateway too ?

Reply:
I moved the functions that used the command store to the command store as methods.

Reply:
The command store is required by the gateway fixture, so it will always be initialized together with the mock gateway.